data/__init__.py

```diff
@@ -54,24 +54,26 @@ def create_dataset(opt):
         >>> from data import create_dataset
         >>> dataset = create_dataset(opt)
     """
-    data_loader = CustomDatasetDataLoader(opt)
+    data_loader = CustomDatasetDataLoader(opt, val_eval=val_eval)
     dataset = data_loader.load_data()
     return dataset
 
 
 class CustomDatasetDataLoader():
     """Wrapper class of Dataset class that performs multi-threaded data loading"""
 
-    def __init__(self, opt):
+    def __init__(self, opt, val_eval=False):
```

Comment:
Would be nice to update the class documentation to include val_eval, and when it should be True or False